README.md

```diff
@@ -49,6 +49,9 @@ The shiniest party lights you have ever seen
 Nikolay's latest fancy plots 
 
 ## Blacklist (list)
+
+Thomas J.
+Gerrit L.
```

Comment:
Gerrit is cool, let's invite him

Reply:
Fair enough. Let's hope TJ is not on GitHub.

Reply:
The bosses watch you